### ISSUE TITLE
[Adaptive Cards - Schema Explorer] - 'Publish Adaptive Card schema' text present on the page is visually appearing as heading but not defined as heading programmatically.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9303]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/
OS Version: 25H2 (OS Build 26200.5761)
Browser Version: Version 145.0.3734.1 (Official build) dev (64-bit)
Tool: Accessibility Insights for Web Tool-> 'Ad hoc tools'-> 'Headings', Developer Tool

**Repro Steps:**

1. Open the above URL on the Edge Dev browser.
2. Navigate to the 'Schema Explorer' present on the header section of the page using tab key and activate using enter key.
3. Now navigate to the 'Publish Adaptive Card schema' text present on the page using tab key.
4. Turn on 'Headings' toggle control of Accessibility Insights for Web tool or press F12 key to open developer tool and inspect the code.
5. Verify that, 'Publish Adaptive Card schema' text present on the page is defined inside the heading or not.

**Actual Result:**
'Publish Adaptive Card schema' text present on the page is visually appearing as heading but not defined as heading programmatically.

**Expected Result:**
'Publish Adaptive Card schema' text present on the page is visually appearing as heading and it should be defined inside the h2 heading programmatically.

**User Experience:**
The text, visually appearing as heading is not defined under heading tag impacts the screen reader users in understanding context of the section defined under that heading as the information and relationships that are implied by visual formatting is not preserved programmatically.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships.html

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![Image](https://github.com/user-attachments/assets/8efd1771-0341-4dfd-bc0c-582b4f09f3c5)

### Card JSON

```JSON
NA
```
